scripts/zones/Crawlers_Nest_[S]/npcs/Tucker.lua

```diff
@@ -5,9 +5,11 @@
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
+    player:startEvent(14) -- doesn't want your trade dialog
 end
 
 entity.onTrigger = function(player, npc)
+    player:startEvent(15) -- rolanberry history dialog
```

Comment:
This can probably move to default actions, but there's a run-up for other dialogues.  I want to say events 4 (one-time) and 5

Reply:
> This can probably move to default actions, but there's a run-up for other dialogues. I want to say events 4 (one-time) and 5

I was JUST looking at that.

Reply:
As we discussed that this would require a variable since one of these require a one time only event and we don't want to leave a forever variable in the system. So we'll move past this, I've also adjusted the event 14 in as a else catchall on trade during the quest as you verified on retail as it's a trade deny after the fact. Recommited. Lets get this working for those who need the vellum.